### ISSUE TITLE
Ajout de fonctionnalités - Un arbre de répertoires (comme Google Drive ou Samepage) pour une organisation

### ISSUE THREAD

**Issue by JeanPaulGrange** [assemblee-virtuelle/semapps#486]
En tant que gestionnaire d'une organisation je souhaite pouvoir créer des répertoires et sous répertoires afin de fournir aux utilisateurs de base une vie claire de l'organisation entre les différents niveaux : internationaux, nationaux, régionaux, départementaux, locaux.
Exemple : structure de dossiers sous Google Drive ou Samepage.io
Je souhaite pouvoir donner accès à des personnes de mon organisation et à des personnes d'autres organisations à ces répertoires (si possible aussi à des groupes)
Je souhaite permettre à des personnes d'avoir le droit de créer des sous répertoires dans un dossier donner.
Je souhaite que les utilisateurs puissent voir qui a accès à un répertoire où ils ont accès.

**Comment by GuillaumeAV** [comment]
Bonjour @JeanPaulGrange 

Héhé !
On a eu une discussion proche sur l'issue #474 ... On y interroge la pertinence de créer des arborescences ... 

Quelques compléments par rapports aux éléments dans l'issue 474 : 
- Les bases de connaissances sémantiques sont basées sur des architectures graphes beaucoup plus fidèles à la réalité. 
Prenons l'exemple d'un document qui concernerait le financement de la fonctionnalité template de gare centrale sur SemApps. 
Dans une architecture arborescente, où devrait-on le classer : Dans le dossier SemApps/Financement ; Dans le dossier AssembléeVirtuelle/Financements ; Dans SemApps/Fonctionnalité ; Dans le dossier AgoraArchipel/ProjetSemApps ; 

Le web sémantique (et l'approche qu'on privilégie avec SemApps) permet de lier le document qui concernerait le financement de la fonctionnalité template de gare centrale sur SemApps, à SemApps, Assemblée Virtuelle, Financement, Fonctionnalité et ProjetSemApps. Et donc de retrouver ce doc à partir de ces multiples entrées. C'est au final proche de la logique des tags qui sont là pour naviguer par delà les répertoires arborescents.

Sur les contrôles d'accès : On est en train de les implémenter : Pouvoir accéder à une donnée et écriture et ou en lecture, en respectant les standards ouverts en la matière : De sorte qu'en implémentant ces standards (identification, authentification, contrôle d'accès notamment),  on puisse donner des accès à des utilisateurs situés sur d'autres plateformes respectant les mêmes standards 
L'enjeu étant de rendre interopérable les plateformes pour favoriser le développement de réseaux sociaux distribués / décentralisés, d'archipels de plateformes autonomes et reliées. 
Voir pour cela le projet SOLID : https://blog.orgtech.fr/un-avenir-solid/

**Comment by srosset81** [comment]
Je suis d'accord qu'un affichage sous forme d'arborescence peut avoir du sens dans certains cas de figures, même si la navigation en graph est plutôt favorisée dans SemApps. Rien ne nous empêche d'ajouter ce type de visualisation des données.

**Comment by GuillaumeAV** [comment]
En effet on pourrait, avec [SKOS ](https://fr.wikipedia.org/wiki/Simple_Knowledge_Organization_System)notamment, qui est déja importé dans PAIR  :) Mais de mon point de vue, cette option doit être creusée après avoir exploité les possibilités de classification permises par l'ontologie .. 
@JeanPaulGrange quand ce sera le moment ca vaudra le coup qu'on te fasse une démo de la manière dont on peut structurer les données avec des outils sémantiques. Le web sémantique permet de sortir des logiques de classification documentaire, mais il faut une démo pour cela ...

**Comment by JeanPaulGrange** [comment]
OK pour la démo quand vous serez prêts !
